### ISSUE TITLE
Will not work when connecting to Drupal

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#10]
```
What's the problem?

No matter how I try to connect to a Drupal Blog (which accepts Wordpress 
and Moveable Type API's) I get this error message:
Well, this is embarrassing...

Sigh... ScribeFire couldn't get the information it needed about your blog. 
Helpfully, your blog returned this message:

Server error. Requested method wp.getUsersBlogs not specified.



What version of ScribeFire for Chrome are you running?
 scribefire-0.1.0.0.crx
```

-----
Original issue reported on code.google.com by `trha...@gmail.com` on 15 Apr 2010 at 5:59

**Comment by GoogleCodeExporter** [comment]
```
What is the URL of your Drupal blog?
```

-----
Original comment by `cfinke` on 20 Apr 2010 at 2:57
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
http://althusius.net

This issue seems to have been corrected in scribefire-0.1.1.0.crx beta released 
April 
16, 2010

Thanks!
```

-----
Original comment by `trha...@gmail.com` on 21 Apr 2010 at 1:12

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 25 Apr 2010 at 3:51
* Changed state: **Fixed**
